test/integration/good/code-gen/glm.stan

```diff
@@ -0,0 +1,62 @@
+data {
```

Comment:
This is silly, but can we name this `optimize_glm.stan` for now? The reason is very silly - I'd like for its output to be after the mother models' output in cpp.expected because I would like to be able to verify that the mother model's generated cpp hasn't been changed, and the globbing is happening in alphabetical order.

Reply:
That seems reasonable.